frontend/src/metabase/ui/components/buttons/Button/Button.styled.tsx

```diff
@@ -32,6 +37,7 @@ export const getButtonOverrides = (): MantineThemeOverride["components"] => ({
               marginLeft: 0,
             },
           },
+          "&:active": animate ? "" : { transform: "none" },
```

Comment:
it's looking like you're using this to disable animations? does the empty string break existing animations?